src/modules/homepage/index.tsx

```diff
@@ -0,0 +1,175 @@
+import {
+  Avatar,
+  Box,
+  Container,
+  IconButton,
+  Stack,
+  Typography,
+} from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { BottomNavigation } from '../../shared-components/BottomNavigation';
+import personImage from '../../assets/illustrations/05.png';
+import { ArrowDropDown } from '@mui/icons-material';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+import ArrowDownwardIcon from '@mui/icons-material/ArrowDownward';
+import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward';
+import LocalMallIcon from '@mui/icons-material/LocalMall';
+import EventRepeatIcon from '@mui/icons-material/EventRepeat';
+import RestaurantIcon from '@mui/icons-material/Restaurant';
+import { colors } from 'src/styles/colors';
+import { Regular3 } from '../../shared-components/Typography/Regular3/index';
+import { IncomeExpenseCard } from './IncomeExpense';
+import { SpendFrequency } from './SpendFrequency';
+import { RecentTransaction } from '../../shared-components/Bars/RecentTransaction';
+import { TransactionCard } from '../../shared-components/Card/TransactionCard';
+import { TimePeriodTabs } from '../../shared-components/Navigation/Tabs/TimePeriodTabs';
+
+const StyledPageContentDiv = styled('div')({
+  overflow: 'auto',
+  maxHeight: 'calc(100vh - 70px)',
+});
+
+const StyledBottomNavigationContainer = styled('div')({
+  position: 'sticky',
+  bottom: 0,
+  left: 0,
+  right: 0,
+  width: '375px',
+  zIndex: '1000',
+  backgroundColor: '#fff',
+});
+
+const StyledRectangle = styled(Box)({
+  textAlign: 'center',
+  width: '375px',
+  height: '312px',
+  gap: '0px',
```

Comment:
Remove gap, it has no meaning if the element is not a flex/grid container

Reply:
Removed